examples/2016-10-31/custom_domains_with_route53/README.md

```diff
@@ -19,3 +18,5 @@ $ sam deploy \
     --stack-name my-new-stack \
     --capabilities CAPABILITY_IAM
 ```
+
+Curl to the endpoint "http://example.com/home/fetch" should hit the Api.
```

Comment:
`home` is the base path and `fetch` is the api path, correct? That's what it looks like looking at the example template, just want to make sure I understand this.